test/test_torch.py

```diff
@@ -6698,7 +6698,7 @@ def test_signed_shift(self, device, dtype):
         a = torch.tensor([-10, 10], device=device, dtype=dtype)  # [11...1110110, 1010]
         expected_l = torch.tensor([-40, 40], device=device, dtype=dtype)  # [11...11011000, 101000]
         self.assertEqual(a << 2, expected_l)
-        expected_r = torch.tensor([torch.iinfo(dtype).max - 4, 5], device=device, dtype=dtype)  # [1111...111011, 101]
+        expected_r = torch.tensor([-5, 5], device=device, dtype=dtype)  # [1111...111011, 101]
```

Comment:
can you also add `compare_with_numpy` test (look in test_torch for example usage) to make sure that our behavior agrees with numpy?